MapboxCoreNavigation/CLLocation.swift

```diff
@@ -114,17 +114,27 @@ extension CLLocation {
             return stepCoordinates
         }
         
-        return nearbyCoordinates
+        return legProgress.nearbyCoordinates
     }
     
+    func projectedDistance(for distanceRemaining: CLLocationDistance, previousSpeeds: [CLLocationSpeed]) -> CLLocationDistance {
+        var defaultDeadReckonDistance = speed * RouteControllerDeadReckoningTimeInterval
+        
+        // If the user is reducing in speed, start to dead reckon less
```

Comment:
Should we also start dead-reckoning more if the user starts accelerating, to avoid undershooting?